build/README.md

```diff
@@ -17,3 +17,4 @@ In our Docker Hub Repository we have the following images:
 - `mattermost/mattermost-build-server:dec-7-2018` which is based on Go 1.11 you can use for MM versions <= `5.8.0`
 - `mattermost/mattermost-build-server:feb-28-2019` which is based on Go 1.12 you can use for MM versions >= `5.9.0` <= `5.15.0`
 - `mattermost/mattermost-build-server:sep-17-2019` which is based on Go 1.12.9 you can use for MM versions >= `5.16.0`
+- `mattermost/mattermost-build-server:20200322_golang-1.14.1` which is based on Go 1.14.1 you can use for MM versions >= `5.23.0`
```

Comment:
Just to double check: If go 1.14.1 should be used for `5.23.0`, this PR has to be cherry-picked one `5.22.0` is cut, right?

Reply:
No, we don't intend for 1.14.1 to be used in 5.23. Only from 5.24 onwards.